llm/tools/ipfs.test.ts

```diff
@@ -0,0 +1,23 @@
+import { describe, test, expect } from "bun:test";
+import { upload_to_ipfs } from "./ipfs";
+
+describe("IPFS Upload", () => {
+  test("should upload content successfully", async () => {
+    const name = "hero.json";
+    const content = {
+      name: "Hero",
+      description: "Hero is a hero",
+      image: "https://example.com/hero.png",
+      attributes: [
+        {
+          attack: "Hero",
+          value: "Hero",
+        },
+      ],
+    };
+
+    const upload = await upload_to_ipfs(name, content);
+
+    expect(upload.id).toBeDefined();
```

Comment:
is it possible to do a query via `fetch()` to that public address?

Reply:
https://github.com/Moonsong-Labs/akton25-cobalt/pull/1/commits/b470eab19890f982eb50edf30723e10f8d734c69